integration-cli/docker_cli_exec_test.go

```diff
@@ -25,9 +25,7 @@ func (s *DockerSuite) TestExec(c *check.C) {
 
 	out, _ := dockerCmd(c, "exec", "testing", "cat", "/tmp/file")
 	out = strings.Trim(out, "\r\n")
-	if out != "test" {
-		c.Errorf("container exec should've printed test but printed %q", out)
-	}
+	c.Assert(out, checker.Equals, "test")
```

Comment:
I thinks this one is good to remove because of the content itself, but we should better to keep the  comment in other case which the content gives out some other information :)